src/app/qgsprojectlayergroupdialog.cpp

```diff
@@ -135,7 +132,7 @@ QString QgsProjectLayerGroupDialog::selectedProjectFile() const
 
 bool QgsProjectLayerGroupDialog::isValid() const
 {
-  return nullptr != mTreeView->layerTreeModel();
+  return static_cast< bool >( mTreeView->layerTreeModel() );
```

Comment:
Why is this changed?

Reply:
I'm not sure (it's from a Nyall's commit), but I think it's much clearer/explicit this way.